mabwiser/linear.py

```diff
@@ -90,50 +90,47 @@ def predict(self, x):
         # Calculate default expectation y = x * b
         return np.dot(x, self.beta)
 
-    def _scale_predict_context(self, x):
+    def _scale_predict_context(self, x: np.ndarray):
         if not hasattr(self.scaler, 'scale_'):
             return x
 
-        # Reshape 1D array to 2D
-        x = x.reshape(1, -1)
-
         # Transform and return to previous shape. Convert to float64 to suppress any type warnings.
-        return self.scaler.transform(x.astype('float64')).reshape(-1)
+        return self.scaler.transform(x.astype('float64'))
 
 
 class _LinTS(_RidgeRegression):
 
-    def predict(self, x):
-
+    def predict(self, x: np.ndarray):
         # Scale
         if self.scaler is not None:
             x = self._scale_predict_context(x)
 
         # Randomly sample coefficients from multivariate normal distribution
         # Covariance is enhanced with the exploration factor
-        beta_sampled = self.rng.multivariate_normal(self.beta, np.square(self.alpha) * self.A_inv)
```

Comment:
what happened here?

Reply:
The above line of code draws random samples from a multivariate normal distribution for each context individually. So this line was modified to generate the random samples for all the contexts in one single go.

Reply:
These are good commentary (ex; generate random samples for all contexts in one single go) ee if you can add a bit of this as code comments, if not already